hs_core/views/resource_folder_hierarchy.py

```diff
@@ -353,8 +353,8 @@ def data_store_file_or_folder_move_or_rename(request):
     except PermissionDenied:
         return HttpResponse('Permission denied', status=status.HTTP_401_UNAUTHORIZED)
 
-    src_path = request.POST.get('source_path', None)
-    tgt_path = request.POST.get('target_path', None)
+    src_path = request.POST.get('source_path', request.data.get('source_path'))
+    tgt_path = request.POST.get('target_path', request.data.get('target_path'))

```

Comment:
My test covers missing params, and it throws a 400 instead of a 500, so I think it's ok